jobs/.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
Instead of adding one `.gitignore` per directory you can just update the one at the top level and add `**.DS_Store` to ignore all files in all directory matching **.DS_Store**